prow/cmd/sidecar/main.go

```diff
@@ -31,15 +34,23 @@ func main() {
 	logrus.SetLevel(logrus.DebugLevel)
 
 	o := sidecar.NewOptions()
+
 	if err := options.Load(o); err != nil {
 		logrus.Fatalf("Could not resolve options: %v", err)
 	}
 
+	logFile, err := ioutil.TempFile("", "sidecar-logs*.txt")
```

Comment:
nit: Since we're only going to create one file with this naming scheme we don't need the `*`. Dropping that will also give us deterministic file names which could be handy for testing and debugging.

Reply:
per the docs, tempFile generates a random string and appends it to the specified prefix, theres no flag or anything to toggle this off. The other work around i see is making a empty temp dir, then creating a file inside the dir. Then removing the dir after. Is this ok?

Reply:
Oh right, nevermind this comment then. As long as we are using temp files/dirs we won't have a deterministic filepath.